### PR TITLE
Compacted RUN statements

### PR DESCRIPTION
**Pull request by rporres**
To avoid having too many layers

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,35 +1,26 @@
...
-RUN apk add --no-cache python3 && \
+RUN apk update && \
+    apk add --no-cache go && \
+    apk add --no-cache nodejs && \
+    echo "alias ll='ls -la'" >> ~/.bashrc && \
+    ln -s /root/.yarn/bin/yarn /bin/yarn && \
+    curl -o- -L https://yarnpkg.com/install.sh | bash && \
+    echo ok > /var/www/public/healthcheck && \
+    apk --update add openjdk7-jre && \
+    apk --update add openjdk7-jre && \
```
Comment (dpordomingo) [inline Dockerfile]: two times the very same?

**Review by dpordomingo — CHANGES_REQUESTED** [review]
Maybe a typo in the original Dockerfile that you reproduced?

... Could we have visual groups for the semantically different things?

- 1st. Prepare the machine with docs dependencies
- 2nd. Install java and sbt dependencies
- 3rd. Install python3 dependencies

**Review by dpordomingo — APPROVED** [review]
Maaaaany thanks.
Ready to merge?

**Comment by dpordomingo** [comment]
Maybe an squash would be great :smile:

**Comment by rporres** [comment]
Let me squash this for you

**Comment by rporres** [comment]
Ready for you! @dpordomingo

**Comment by dpordomingo** [comment]
many thanks

**Pull request merged**